src/event/data.rs

```diff
@@ -46,7 +46,7 @@ impl Data {
 }
 
 pub(crate) fn is_json_content_type(ct: &str) -> bool {
-    ct == "application/json" || ct == "text/json" || ct.ends_with("+json")
```

Comment:
Why change it over here?

Reply:
because this method was rejecting content types like `application/json; charset=utf-8`

Reply:
Ahhh, i see